fixtures/integrations/jira/stubs/unexpected_user.json

```diff
@@ -0,0 +1,16 @@
+{
+  "active": true,
+  "avatarUrls": {
+    "24x24": "https://secure.gravatar.com/avatar/0123456789abcdef0123456789abcdef?d=https%3A%2F%2Favatar-management--avatars.us-west-2.prod.public.atl-paas.net%2Finitials%2FAB-6.png",
+    "16x16": "https://secure.gravatar.com/avatar/0123456789abcdef0123456789abcdef?d=https%3A%2F%2Favatar-management--avatars.us-west-2.prod.public.atl-paas.net%2Finitials%2FAB-6.png",
+    "48x48": "https://secure.gravatar.com/avatar/0123456789abcdef0123456789abcdef?d=https%3A%2F%2Favatar-management--avatars.us-west-2.prod.public.atl-paas.net%2Finitials%2FAB-6.png",
+    "32x32": "https://secure.gravatar.com/avatar/0123456789abcdef0123456789abcdef?d=https%3A%2F%2Favatar-management--avatars.us-west-2.prod.public.atl-paas.net%2Finitials%2FAB-6.png"
+  },
+  "displayName": "Saif Hakim",
+  "accountType": "atlassian",
+  "timeZone": "America/Los_Angeles",
+  "locale": "en_US",
+  "emailAddress": "",
+  "self": "https://example.atlassian.net/rest/api/2/user?accountId=012345:00000000-1111-2222-3333-444444444444",
+  "unexpectedId": "012345:00000000-1111-2222-3333-444444444444"
```

Comment:
I copied `fixtures/integrations/jira/stubs/user.json` and edited the `userId` field so that it can be used in `src/sentry/tests/sentry/integrations/jira/test_utils.py`'s `test_unexpected_id`. `build_user_choice` is now broken out into a Jira Cloud version and a Jira Server version - previously you had to pass in a `user_id_field` that was only used because Jira Cloud has one field, and Jira Server has another.